src/server/enrollment-system.js

```diff
@@ -63,35 +63,97 @@ function veteranToPersonInfo(veteran) {
   };
 }
 
+function zeroPadNumber(number, padding) {
+  return (new Array(padding + 1).join('0') + number).slice(-padding);
+}
+
+function formDateToESDate(dateObject) {
+  if (dateObject.month >= 1 && dateObject.month <= 12 &&
+    dateObject.day >= 1 && dateObject.month <= 31 && // TODO: how robust does this need to be? Form validates as well.
+    dateObject.year > 1) {
+    return `${zeroPadNumber(dateObject.month, 2)}/${zeroPadNumber(dateObject.day, 2)}/${zeroPadNumber(dateObject.year, 4)}`;
+  }
+  return undefined;
+}
+
+function makeServiceBranch(serviceBranch) {
+  // NOTE the return codes are from VHA Standard Data Service (ADRDEV01) Service Branch List
```

Comment:
Move this note to a function comment?  Something like:

// Converts serviceBranch from the values in Veteran resource to VHA Standard Data Service Codes.
//
// Codes are from the VHA Standard Data Service Codes (ADRDEV01) Service Branch List.
//   http://vaausesrapp80.aac.va.gov:7404/ds/List/ServiceBranch